kobiton-rest-api/get-devices-info/README.md

```diff
@@ -0,0 +1,140 @@
+# Getting device(s) information
+
+## Table of contents
+- [A. Overview](#a-overview)
+- [B. Preparations](#b-preparations)
+  - [1. Prerequisites](#1-prerequisites)
+  - [2. Getting Kobiton Username and API Key](#2-getting-kobiton-username-and-api-key)
+  - [3. Getting Kobiton Group ID](#3-getting-kobiton-group-id)
+- [C. Samples](#c-samples)
+
+## A. Overview
+This guideline will provide instructions and sample use cases for utilizing Kobiton REST API `/devices` endpoint.
+
+## B. Preparations
+### 1. Prerequisites
+- cURL installed.
+  - `Windows`: Visit https://curl.haxx.se/windows/ for instructions on how to download and install cURL on Windows.
+  - `Linux`: cURL installed by default in most Linux distributions. If not, please refer to corresponding distro instructions on how to download and install cURL.
+  - `macOS`: cURL installed by default.
+
+### 2. Getting Kobiton Username and API Key
+Kobiton Username and API key are required for authenticating with Kobiton API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script for Kobiton` section on [our blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 3. Getting Kobiton Group ID
+If your account is in an organization and assigned to one or more groups, you will need to get the group ID.
+
+Follow instructions at https://docs.kobiton.com/organization-management/automation-for-groups/default-group-setting/ to get your group ID.
+
+## C. Samples
+
+In this section, we will be providing sample use cases for `/devices` endpoint using bash commands with cURL.
+
+- When execute the commands provided in this section, remember to replace these variables with your values:
+  - `<YOUR_KOBITON_USERNAME>`: Your Kobiton Username.
+  - `<YOUR_KOBITON_API_KEY>`: Your Kobiton API Key.
+  - `<YOUR_GROUP_ID>`: Your group ID.
+
+- Expected output for all samples in this section:
+```javascript
+{"privateDevices":[...],"favoriteDevices":[...],"cloudDevices":[...]}
+```
+
+For instance: 
+
+Getting detailed information of all devices:
+
+- User has no organization and no group:
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
+Example:
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices' \
+  -H 'Accept: application/json' \
+  -u kobitondemo:4bbaa732-d920-4de2-a93b-43f2369f478d
+```
+
+- User has organization and is in group(s):
```

Comment:
I think you should:
1. Cover one more case: "User in the organization and not in group" 
2. Describe what happens when users in group(s) don't specify the groupID? 

Note: apply for sections below, my suggestion: create the specific section highlight these cases.